_search-plugins/ubi/sql-queries.md

```diff
@@ -0,0 +1,381 @@
+---
+layout: default
+title: Sample UBI SQL queries
+parent: User behavior insights
+has_children: false
+nav_order: 7
+---
+
+# Sample UBI SQL queries
+
+You can run the following sample SQL queries in an OpenSearch Dashboards [Query Workbench]({{site.url}}{{site.baseurl}}/dashboards/query-workbench/) sandbox application:  
+
+[http://chorus-opensearch-edition.dev.o19s.com:5601/app/opensearch-query-workbench](http://chorus-opensearch-edition.dev.o19s.com:5601/app/opensearch-query-workbench)
```

Comment:
This link is unreachable to me. Also, please provide descriptive text for the link (where is the user going?) For example, `See the [sample application](link)`.

Reply:
None of these work for me :(

Reply:
@kolchfa-aws, could you try pinging chorus-opensearch-edition.dev.o19s.com first?  If that doesn't work, could you try pinging 159.89.182.117 ?

```
 ping chorus-opensearch-edition.dev.o19s.com

Pinging chorus-opensearch-edition.dev.o19s.com [159.89.182.117] with 32 bytes of data:
Reply from 159.89.182.117: bytes=32 time=87ms TTL=44
Reply from 159.89.182.117: bytes=32 time=87ms TTL=44
Reply from 159.89.182.117: bytes=32 time=86ms TTL=44
Reply from 159.89.182.117: bytes=32 time=85ms TTL=44
```

Reply:
I just tried it...   And it loaded.   I suspect that your corporate firewall may be blocking it...   
<img width="478" alt="image" src="https://github.com/opensearch-project/documentation-website/assets/22395/2ca9a968-6ed0-4ccd-b415-177d588e4017">

Reply:
Ping works but accessing the website through the browser or via curl doesn't.

Reply:
That said, the link checker is not complaining so it's probably the firewall.